scripts/Makefile.toolchain.xcc

```diff
@@ -1,5 +1,3 @@
-XTENSA_ISS_CYCLES_LIMIT := 1000000000 # Simulate 10^9 cycles.
```

Comment:
I would preferred to keep this as users don't always use sanitychk to run their applications.
Also the turbo mode does not fit some very strict timing tests/applications so I would expect some advanced users (mostly companies) to prefer using the cycle accurate simulations.